doc/md/guides/deploying/init.md

```diff
@@ -0,0 +1,160 @@
+---
+id: k8s-init-containers
+slug: /guides/deploying/k8s-init-container
+title: Deploying schema migrations to Kubernetes with Init Containers
+---
+
+In [Kubernetes](https://kubernetes.io), [Init Containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
+are specialized containers that run before app containers in a Pod. Init containers
+can contain utilities or setup scripts not present in an app image. 
+
+Init containers can be utilized to run schema migrations with Atlas before the
+application loads. Because init containers can use a container image different
+from the application, developers can use a [purpose-built image](image.md) that
+only contains Atlas and the migration scripts to run them.  This way, less 
+can be included in the application runtime environment, which reduces
+the attack surface from a security perspective. 
+
+Depending on an application's [deployment strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy),
+multiple replicas of an init container may run concurrently.  In the case of
+schema migrations, this can cause a dangerous race condition with unknown outcomes.
+To prevent this, in databases that support advisory locking, Atlas will acquire
+a lock on the migration operation before running migrations, making the
+operation mutually exclusive.
```

Comment:
```suggestion
To prevent this, in databases that support advisory locking, Atlas will acquire
a database lock before running migrations, making the operation mutually exclusive.
```